docs/pages/installation/connect-requirements.md

```diff
@@ -0,0 +1,109 @@
+
+#### Additional considerations
+
+The choice of the way you expose Liqo services to remote cluster may not be trivial in some cases.
+Here we list some additional notes you should consider in your choice:
+
+* **NodePort service**: although a NodePort service can be used to expose the Auth service and the Network gateway, often the IP addresses of the nodes are configured with private IP addresses, hence not being suitable for connections originated from the Internet. This happens rather often in production clusters, and on public clusters as well.
+* **Ingress controller**: in case the Auth service is exposed through an Ingress, you should remember that, by default, the Auth service uses the TLS protocol. Hence, either you configure your Ingress controller to connect to backend services with TLS as well, or you disable TLS on the Auth Service.
+* **API Server behind NAT**: in case the cluster is behind NAT, the IP address/port of the API server (i.e., `apiServer.address` in `values.yaml`) must be set as the _public_ <IP/port> tuple. This is not required for the Auth service, as the liqoctl command used to connect to the remote cluster 
+* **Auth service behind NAT**: in case your cluster is behind NAT, the `liqoctl` command used by the remote cluster to peer with you must use the _public_ <IP/port> tuple of the Auth service.
+
+Finally, currently Liqo supports only scenarios in which _one_ of the cluster is **behind NAT**; network connectivity cannot be established in case both clusters are behind NAT. This limitation should be addressed in future releases.
+
+
+### Clusters in the same LAN
+
+This represents a very particular scenario supported by Liqo, in which clusters (K3s or K8s) in the same LAN can rely on the mDNS-based discovery mechanism implemented by Liqo to identify all the other clusters present on the same LAN segment.
+This enables easy cluster setup (and peering) e.g., in case of rapid prototyping.
+
+![](/images/scenarios/on-prem-to-on-prem.svg)
+
+Although other alternatives are possible, we suggest the easiest configuration in this case, which relies on the following options:
+
+|           | Cluster A (On-LAN)  | Cluster B (On-LAN) |
+| --------- | ------------------- | ------------------- |
+| **Auth service** |  NodePort | NodePort |
+| **Network gateway** | NodePort | NodePort |
+| **API server** | Exposed | Exposed |
+
+
+### Network firewalls
+Once Liqo has been installed, you may have to configure existing firewalls to enable remote clusters to contact the three endpoints (`liqo-auth`, `liqo-gateway`, API server) that need to be publicly accessible in the peering phase.
+
+To know the network parameters (i.e., <IP/port>) used by `liqo-auth` and `liqo-gateway`, you can use standard Kubernetes commands (e.g., `kubectl get services -n liqo-namespace`), while you should know the <IP/port> used by your API server since it is written in your `kubeconfig` file.
```

Comment:
```suggestion
To know the network parameters (i.e., <IP/port>) used by `liqo-auth` and `liqo-gateway`, you can use standard Kubernetes commands (e.g., `kubectl get services -n liqo`), while you should know the <IP/port> used by your API server since it is written in your `kubeconfig` file.
```